src/main/java/org/candlepin/insights/inventory/ConduitFacts.java

```diff
@@ -20,21 +20,39 @@
  */
 package org.candlepin.insights.inventory;
 
+import org.hibernate.validator.constraints.Length;
+
 import java.util.List;
 import java.util.UUID;
 
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Positive;
+
 /**
  * POJO that holds all facts scoped for collection by the conduit.
  */
 public class ConduitFacts {
     private UUID subscriptionManagerId;
     private UUID biosUuid;
-    private List<String> ipAddresses;
+
+    @NotEmpty
+    private List<@Length(min = 7, max = 45) String> ipAddresses;
+
+    @Length(min = 1, max = 255)
     private String fqdn;
-    private List<String> macAddresses;
+
+    @NotEmpty
+    private List<@Length(min = 1, max = 23) String> macAddresses;
```

Comment:
Should we be enforcing @NotEmpty here? We should be only sending what we know.

Reply:
Similar to the previous comment: https://github.com/dehort/insights-host-inventory/blob/input_validation/swagger/api.spec.yaml#L394

Reply:
As discussed in scrum, we should at least remove the not null constraint.